modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BreadcrumbDisplayContext.java

```diff
@@ -58,15 +66,54 @@ public Map<String, Object> getProps() throws Exception {
 					"symbolLeft", "cog"
 				),
 				JSONUtil.put(
+					"href",
```

Comment:
@oliv-yu, you can use the permissionsTag to achieve this, something like this: 

PermissionsURLTag.doTag(
			StringPool.BLANK, DepotEntry.class.getName(), group.getName(), null,
			String.valueOf(depotEntry.getDepotEntryId()),
			LiferayWindowState.POP_UP.toString(), null,
			liferayPortletRequest.getHttpServletRequest());

I think you have all variables to achieve it, we can take a look together if you want

Reply:
Sounds good! `PermissionsURLTag.doTag` worked for me when I did this (let me know if there's any issues with it):


```
					PermissionsURLTag.doTag(
						StringPool.BLANK, DepotEntry.class.getName(),
						group.getName(), null,
						String.valueOf(group.getClassPK()),
						LiferayWindowState.POP_UP.toString(), null,
						_httpServletRequest)
```